ensime_shared/debugger.py

```diff
@@ -65,3 +65,21 @@ def debug_backtrace(self, args, range=None):
             "typehint": "DebugBacktraceReq",
             "threadId": self.debug_thread_id,
             "index": 0, "count": 100})
+
+    def debug_step(self, args, range=None):
+        self.log("debug_step: in")
+        self.send_request({
+            "typehint": "DebugStepReq",
+            "threadId": self.debug_thread_id})
+
+    def debug_step_out(self, args, range=None):
+        self.log("debug_step_out: in")
+        self.send_request({
+            "typehint": "DebugStepOutReq",
+            "threadId": self.debug_thread_id})
+
+    def debug_next(self, args, range=None):
+        self.log("debug_next: in")
```

Comment:
Since the logging is merged these will need to be `self.log.debug` not to break.